Formula/lua.rb

```diff
@@ -45,7 +45,8 @@ class Lua < Formula
   # sigaction provided by posix signalling power patch
   if build.with? "sigaction"
     patch do
-      url "http://lua-users.org/files/wiki_insecure/power_patches/5.2/lua-5.2.3-sig_catch.patch"
+      # url "http://lua-users.org/files/wiki_insecure/power_patches/5.2/lua-5.2.3-sig_catch.patch"
+      url "https://gist.githubusercontent.com/anonymous/99614c27bcac2aa86181fabe78c85915/raw/7abdf0234ffd38fc9094078d8128938f5e2accd4/lua-5.2.3-sig_catch.patch"
```

Comment:
> Of all the names suggested so far formula-patches is probably my favorite (no plural “e” for readability and I believe also grammar).

`formula-patches` it is. OK with you @DomT4?

> A rename will probably just shift confusion from one group of people to another, at the expense of those who have already familiarized themselves with it and will be surprised to no longer find it in our repository list.

Those who search for it will see it with the a similar name and all previous links redirect.